CHANGELOG.md

```diff
@@ -5,6 +5,52 @@ Documentation of all notable changes to the **Fizzy** project.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+## [0.3.0] — Unreleased
+
+This main focus for this release is to implement every WebAssembly validation rule from the specification.
+
+It passes a large part of the [official test suite (spectest 1.0)]:
+  - 4481 of 4490 binary parser and execution tests,
+  - 942 of 942 validation tests,
+  - 6381 skipped due to containing floating-point instructions or testing text format parser.
+
+Additionally measurable speed improvements were made for function calls:
+
+TODO: include benchmark
+
+### Added
+
+- Type validation of instructions. [#403](https://github.com/wasmx/fizzy/pull/403) 
+  [#404](https://github.com/wasmx/fizzy/pull/404) [#411](https://github.com/wasmx/fizzy/pull/411)
+  [#414](https://github.com/wasmx/fizzy/pull/414) [#415](https://github.com/wasmx/fizzy/pull/415)
+- Type validation of block results. [#407](https://github.com/wasmx/fizzy/pull/407)
+  [#429](https://github.com/wasmx/fizzy/pull/429)
+- Type validation of branch instructions. [#408](https://github.com/wasmx/fizzy/pull/408)
+- Type validation of select instruction. [#413](https://github.com/wasmx/fizzy/pull/413)
+- Type validation in constant expressions. [#430](https://github.com/wasmx/fizzy/pull/430)
+- Validation of `else` branch in `if` instruction. [#417](https://github.com/wasmx/fizzy/pull/417)
+- Validation of frame types in `br_table` instruction. [#427](https://github.com/wasmx/fizzy/pull/427)
+- Validation of function indices in element section. [#432](https://github.com/wasmx/fizzy/pull/432)
+- Validation of functions' arity. [#433](https://github.com/wasmx/fizzy/pull/433)
+- Validation of alignment for floating point memory instructions. [#437](https://github.com/wasmx/fizzy/pull/437)
+- CI improvements. [#405](https://github.com/wasmx/fizzy/pull/405) [#434](https://github.com/wasmx/fizzy/pull/434)
+- Better unit test coverage. [#381](https://github.com/wasmx/fizzy/pull/381) [#410](https://github.com/wasmx/fizzy/pull/410)
```

Comment:
I would drop this as well. But if to stay, move it to `Changed`.

Reply:
I think it is good keeping this under changed.

Reply:
Moved to changed.